src/timeline.js

```diff
@@ -158,6 +168,12 @@ export default class Timeline extends React.Component {
     this.throttledMouseMoveFunc = _.throttle(this.throttledMouseMoveFunc.bind(this), 20);
     this.mouseMoveFunc = this.mouseMoveFunc.bind(this);
     this.getCursor = this.getCursor.bind(this);
+    this.getStartDate = this.getStartDate.bind(this);
```

Comment:
add comment explaining why we bind; all need to be bound? I didn't pay attention if all are passed as params.

Reply:
Done.